extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -82,6 +82,57 @@ class TwitterPluginFactory(
 }
 ```
 
+### Plugin categories
+There are use-cases where you would like to use another plugin in order to share functionality and/or configuration.
+Plugin Categories to the rescue: this functionality lets you choose a plugin implementing a specific interface into your plugin as a `@PluginProperty` in the configuration screen.
+
+For example, you might want to use a supplier for your tweets:
+```kotlin
```

Comment:
First the interface is defined that includes the required functionality. @PluginCategory() with key is added that can be used to find configurations of that type in the front-end